CHANGELOG.md

```diff
@@ -2,10 +2,9 @@ Changes
 =======
 
 0.13.2, 2016-08-19
-* export_phrases in Phrases model changed. Fixed issue #794 and added test cases in test/test_phrases.py(@AadityaJ,
-[#879](https://github.com/RaRe-Technologies/gensim/pull/879))
```

Comment:
Please don't delete other comments in changelog. 
Also please add this change to the future release 0.13.3